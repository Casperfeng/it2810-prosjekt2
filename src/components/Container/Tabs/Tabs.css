.tab {
  font-size: 20px;
  color: rgb(54, 96, 124);
  background-color: rgb(223, 243, 255);
  border-left: 1px;
  border-right: 1px;
  border-top: 0px;
  border-bottom: 0px;
  border-style: solid;
  border-color: white;
  padding: 10px;
  flex-grow: 1;
}

.tab:hover {
  background-color: rgb(170, 208, 231);
}

.tabs-container {
  display: flex;
  flex-flow: row nowrap;
  width: 100%;
}

.active-tab {
  background-color: white;
}

.active-tab:hover {
  background-color: white;
}
