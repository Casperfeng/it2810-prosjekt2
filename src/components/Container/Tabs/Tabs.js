import React from 'react'
import './Tabs.css';

function Tabs(props) {
  return (
    <div>
      <button onClick={() => props.onTabIndexChanged(0)}>1</button>
      <button onClick={() => props.onTabIndexChanged(1)}>2</button>
      <button onClick={() => props.onTabIndexChanged(2)}>3</button>
      <button onClick={() => props.onTabIndexChanged(3)}>4</button>
    </div>
  );
}

export default Tabs;