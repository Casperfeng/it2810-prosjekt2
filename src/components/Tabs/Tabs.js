import React from 'react'
import './Tabs.css';
import  Container from './Container/Container.js'

function Tabs(props) {
  return (
    <div>
      <button onClick={() => props.onTabIndexChanged(0)}>1</button>
      <button onClick={() => props.onTabIndexChanged(1)}>2</button>
      <button onClick={() => props.onTabIndexChanged(2)}>3</button>
      <button onClick={() => props.onTabIndexChanged(3)}>4</button>
      <Container      
      svg = {props.svg} 
      poem = {props.poem} 
      audio = {props.audio}
      svgIndex = {props.svgIndex}
      poemIndex = {props.poemIndex}
      audioIndex = {props.audioIndex}
      />
    </div>
  );
}

export default Tabs;