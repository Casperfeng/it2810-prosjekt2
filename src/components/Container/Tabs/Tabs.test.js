import React from 'react';
import Tabs from './Tabs';
import renderer from 'react-test-renderer';

it('renders Container correctly', () => {
  const tree = renderer
    .create(<Tabs />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});