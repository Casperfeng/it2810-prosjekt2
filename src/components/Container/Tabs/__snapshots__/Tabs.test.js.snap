// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Container correctly 1`] = `
<div
  className="tabs-container"
>
  <button
    className="tab "
    onClick={[Function]}
  >
    1
  </button>
  <button
    className="tab "
    onClick={[Function]}
  >
    2
  </button>
  <button
    className="tab "
    onClick={[Function]}
  >
    3
  </button>
  <button
    className="tab "
    onClick={[Function]}
  >
    4
  </button>
</div>
`;
